Update documentation on extends (#2362)
executablebooks/mystmd: docs/asides.md

description: Asides provide an easy way to represent content that is only indirectly related to the article's main content, such as in the sidebar or margin.
---

Asides provide an easy way to represent content that is only indirectly related to the article's main content.

> Where supported, MyST will attempt to display an {myst:directive}`aside` _as close as possible_ but separately from the main article, such as in the side-margin.

Reviewer: ```suggestion
Where supported, MyST will attempt to display an {myst:directive}`aside` _as close as possible to_—but separate from—the main article, such as in the side-margin.
```

Reviewer: Could also use commas around the sub-sentence ("possible to, but separate from, ...").

These elements can conflict with the document outline when they are both competing for the margin space (see [#170](https://github.com/jupyter-book/myst-theme/issues/170)).

> There is, yet, no plugin available transforming this directive into a Typst pdf.

Reviewer: ```suggestion
There is, as yet, no plugin available for transforming this directive into Typst.
```

I do have a margin typst snippet, FWIW, if that'd come in handy.
